### DISCUSSION TITLE
Map panel should use 👁️ visibility toggles rather than On/Off switch

### DISCUSSION THREAD

**Discussion by jtbandes** [foxglove/community#553] [Q&A]
Map panel:

<img width="355" alt="image" src="https://user-images.githubusercontent.com/14237/196788936-9a57c7aa-892d-4a19-a0d7-e12b3d549de2.png">

3D panel:
<img width="363" alt="image" src="https://user-images.githubusercontent.com/14237/196789056-612e37b8-c576-4e45-a032-7ce388188d6f.png">

Plot panel:

<img width="369" alt="image" src="https://user-images.githubusercontent.com/14237/196789015-b1fb64d8-4dd6-4840-b2f9-14653bb07cbd.png">

**Comment by defunctzombie** [comment]
@foxymiles you did the initial settings port: https://github.com/foxglove/studio/commit/4e156abb8fb84076d1ccacefb2b45731c730414b

Do you remember why the toggle was picked vs the visibility eye?

**Comment by foxymiles** [comment]
I think it was just because this was one of the very first panels we ported and we hadn't really settled on conventions like this yet.

**Comment by osaro-imohe** [comment]
I'll get this
